p2p/proto/block.proto

```diff
@@ -14,21 +16,25 @@ message Signatures {
 // Note: commitments may change to be for the previous blocks like comet/tendermint
 // hash of block header sent to L1
 message BlockHeader {
-    Hash                      parent_header     = 1;
-    uint64                    number            = 2;
-    google.protobuf.Timestamp time              = 3;   // TODO: see if this needs to be Felt252 or can be converted
-    Address                   sequencer_address = 4;
-    Merkle                    state_diffs       = 5;   //  By order of (contract, key), taking last in case of duplicates.
+    Hash                      hash              = 1;
```

Comment:
What is this hash of?

Reply:
It's the block hash

Reply:
Why do we need a separate hash than the state diff and proof?

Reply:
I don't understand what you mean by proof or its significance.

The main reason for including the Hash was because there are old blocks whose hash we cannot calculate. However, we have decided to provide snapshots of the DB for the peer to start from. Therefore, I will remove it.

Reply:
While providing such snapshots will work in the short term; I don't think its a viable long term solution. This means every node user will have to trust your snapshots without the ability to verify them.

Reply:
Long term, regenesis will happen.